### PR TITLE
Add Snowflake as data source

### PR DESCRIPTION
**Pull request by mosabua**
Wording about Snowflake is from their site.

Merge has to wait until Trino 440 release is out.

### CODE REVIEW

**Comment by mosabua** [comment]
440 is live now - https://trino.io/docs/current/connector/snowflake.html

File: _data/tools.yml [hunk 1]
```diff
@@ -1322,6 +1322,28 @@
       url: https://www.singlestore.com/
     - urltext: SingleStore connector documentation
       url: https://trino.io/docs/current/connector/singlestore.html
+- name: Snowflake
+  anchor: snowflake
+  category: data-source
+  logo: /assets/images/logos/snowflake.png
+  logosmall: /assets/images/logos/snowflake-small.png
+  description: |
+    Snowflake is a Data Coud platform provider. Snowflake easily enables
```
Comment (colebow) [inline _data/tools.yml:1331]:
```suggestion
    Snowflake is a Data Cloud platform provider. Snowflake easily enables
```

File: _data/tools.yml [hunk 2]
```diff
@@ -1322,6 +1322,28 @@
...
+    governed access to near-infinite amounts of data, and cutting-edge tools,
```
Comment (colebow) [inline _data/tools.yml:1332]:
```suggestion
    governed access to near-infinite amounts of data, cutting-edge tools,
```

**Review by colebow — APPROVED** [review]
Two nits then LGTM

**Pull request merged**